core/src/main/java/org/apache/accumulo/core/client/ConditionalWriterConfig.java

```diff
@@ -175,4 +190,32 @@ public String getClassLoaderContext() {
     return this.classLoaderContext;
   }
 
+  private static <T> T merge(T o1, T o2) {
+    if (o1 != null)
+      return o1;
+    return o2;
+  }
+
+  /**
+   * Merge this ConditionalWriterConfig with another. If config is set in both, preference will be
+   * given to this config.
+   *
+   * @param other
+   *          Another ConditionalWriterConfig
+   * @return Merged ConditionalWriterConfig
+   * @since 2.0.0
+   */
+  public ConditionalWriterConfig merge(ConditionalWriterConfig other) {
+    ConditionalWriterConfig result = new ConditionalWriterConfig();
+    result.timeout = merge(this.timeout, other.timeout);
+    result.maxWriteThreads = merge(this.maxWriteThreads, other.maxWriteThreads);
+    result.auths = merge(this.auths, other.auths);
+    if (this.isDurabilitySet) {
```

Comment:
Could do the following so that `merge()` can also be used here.

 * Start durability off as null.
 * Make getDurability return DEFAULT when its null
 * Drop isDurabilitySet